p2p/protocol/holepunch/coordination.go

```diff
@@ -0,0 +1,308 @@
+package holepunch
+
+import (
+	"context"
+	"sync"
+	"time"
+
+	logging "github.com/ipfs/go-log"
+	"github.com/jpillora/backoff"
+	"github.com/libp2p/go-libp2p-core/host"
+	"github.com/libp2p/go-libp2p-core/network"
+	"github.com/libp2p/go-libp2p-core/peer"
+	"github.com/libp2p/go-libp2p/p2p/protocol/holepunch/pb"
```

Comment:
Can you actually name this `pb`? So that the code that uses just does `pb.Foo` instead of `holepuch_pb.Foo`.
It's consistent with how we do things across the stack.

Reply:
done.